prow/config/jobs/istio.yaml

```diff
@@ -108,6 +108,19 @@ jobs:
       - name: TEST_SELECT
         value: "-postsubmit,-flaky,-multicluster"
 
+  - name: integ-operator-controller-tests
+    type: presubmit
+    command: [entrypoint, prow/integ-suite-kind.sh, test.integration.operator.kube.presubmit]
+    requirements: [kind]
+    modifiers: [optional]
```

Comment:
please make it `[optional,hidden]` until the test has stabilized

Reply:
done